README.md

```diff
@@ -38,5 +39,4 @@ $ open http://localhost:9000
 
 ### IntelliJ
 
-1. `$ sbt gen-idea`
-2. Inside IntelliJ, `File/Open...`, choose the root folder to import all the projects (do *not* use `Import Project...` or `Import Module...`)
+IntelliJ integration does not work well with cross projects for now. The IDEA team is working on a fix.
```

Comment:
Yep it's working now with the latest version of the Scala Plugin (v1.3.3). The README.md has been updated by the way https://github.com/vmunier/play-with-scalajs-example/commit/939c7f6d702b8b3f698fc6253413e0b9773a5d38 ;-)